src/libraries/System.Runtime.Loader/tests/AssemblyExtensionsTest.cs

```diff
@@ -27,9 +26,18 @@ public static void ApplyUpdateInvalidParameters()
             Assert.Throws<ArgumentException>(() =>
                 AssemblyExtensions.ApplyUpdate(new NonRuntimeAssembly(), new ReadOnlySpan<byte>(metadataDelta), new ReadOnlySpan<byte>(ilDelta), ReadOnlySpan<byte>.Empty));
 
-            // Tests that this assembly isn't not editable
-            Assert.Throws<InvalidOperationException>(() =>
-                AssemblyExtensions.ApplyUpdate(typeof(AssemblyExtensions).Assembly, new ReadOnlySpan<byte>(metadataDelta), new ReadOnlySpan<byte>(ilDelta), ReadOnlySpan<byte>.Empty));
+            if (PlatformDetection.IsNetCore && PlatformDetection.IsNotMonoRuntime && PlatformDetection.IsArmOrArm64Process)
```

Comment:
```suggestion
            if (PlatformDetection.IsNotMonoRuntime && PlatformDetection.IsArmOrArm64Process)
```
This test only ever runs on .NET Core, so this check is always true.